.htaccess

```diff
@@ -4,4 +4,7 @@ RewriteRule ^download.html$ /downloads/ [R=301,L]
 RewriteRule ^downloads.html$ /downloads/ [R=301,L]
 RewriteRule ^download(/|$) /downloads/ [R=301,L]
 
+RewriteCond %{HTTP_USER_AGENT} libwww-perl.* 
```

Comment:
its a SEO practice to block User-agent Libwww-perl.
Botnet scripts that automatically look for vulnerabilities in your software are sometimes identified as User-Agent libwww-perl. By blocking access from libwww-perl you can eliminate many simpler attacks.